baseplate/context/memcache/__init__.py

```diff
@@ -41,19 +42,30 @@ def pool_from_config(app_config, prefix="memcache.", **kwargs):
             "max_pool_size": config.Optional(config.Integer, default=None),
             "connect_timeout": config.Optional(config.Float, default=None),
             "timeout": config.Optional(config.Float, default=None),
+            "min_compress_length": config.Optional(config.Integer, default=0),
+            "compress_level": config.Optional(config.Integer, default=0),
+            # TODO: bug in Optional prevents this from being unset
```

Comment:
just a note that we need to fix the bug in `config.Optional` before doing an optional boolean config.